packages/core/src/Table/TableCell/TableCell.styles.js

```diff
@@ -0,0 +1,77 @@
+import { colors } from '@dhis2/ui-constants'
+import css from 'styled-jsx/css'
+
+export default css`
+    td,
+    th {
+        padding: 0 12px;
+        font-size: 14px;
+        height: 45px;
+        border: 1px solid transparant;
+        border-bottom: 1px solid ${colors.grey300};
+    }
+    td {
+        background-color: ${colors.white};
+    }
+    th {
+        color: ${colors.grey800};
+        background-color: ${colors.grey200};
+        font-weight: normal;
+        border-right: 1px solid transparent;
+    }
+    th:last-of-type {
+        border-right: 1px solid ${colors.grey300};
+    }
+    td.active {
+        background-color: ${colors.blue100};
+    }
+    td.bordered,
+    th.bordered {
+        border-right: 1px solid ${colors.grey300};
+    }
+    td.bordered:last-child,
+    th.bordered:last-child {
+        border-right: 1px solid transparent;
+    }
+    td.error,
+    th.error {
+        color: ${colors.red700};
+    }
+    td.muted,
+    th.muted {
+        color: ${colors.grey700};
+        font-style: italic;
+    }
+    td.valid,
+    th.valid {
+        color: ${colors.green700};
+    }
+    td.large,
+    th.large {
+        font-size: 16px;
+        height: 60px;
+    }
+    th.fixed {
+        position: sticky;
+        z-index: 1;
+    }
+    :global(tr:last-child) td,
+    :global(tr:last-child) th {
+        border-bottom: 1px solid transparent;
+    }
+    :global(tbody) > :global(tr:hover) > td {
+        background-color: ${colors.blue050};
+    }
+    :global(tbody) > :global(tr:hover) > th {
+        background-color: ${colors.grey300};
+    }
+    :global(tbody) > :global(tr:active) > td {
+        background-color: ${colors.blue100};
+    }
+    :global(tbody) > :global(tr.selected) > td {
+        background-color: ${colors.teal050};
+    }
+    :global(tbody) > :global(tr.selected:hover) > td {
+        background-color: ${colors.teal100};
+    }
```

Comment:
It's not recommended/ would be going against our design-system to place a `TableCell` in a `TableHead`, but it won't produce invalid HTML. So by using these global selectors I have disabled these row-hover effects in a `thead` or `tfoot`.

Interesting side-note: I had some styles here with a global `table` selector as, i.e.: 
```css
:global(table) > :global(tr:hover) > td {}
```
I had included those because I thought a `tr` as a direct descendant of a `table` was valid HTML. This turned out to be incorrect: React will throw an "Invalid DOM Nesting" error. So I removed these `table > tr` styles.

Reply:
*Interesting side-note:*
I had some styles here with a global `table` selector as, i.e.: 
```css
:global(table) > :global(tr:hover) > td {}
```
I had included those because I think a `tr` as a direct descendant of a `table` is valid HTML (I still do). However, React does not agree with me: it will throw an "Invalid DOM Nesting" error. I don't think this is right, but it was very convenient for me because it allowed me to I removed these `table > tr` styles.